### DISCUSSION TITLE
fromitive-blog/coding-interview/2023-07-08-hackerrank-1/

### DISCUSSION THREAD

**Discussion by giscus** [fromitive/fromitive-blog#2] [Q&A]
# fromitive-blog/coding-interview/2023-07-08-hackerrank-1/

HackerRank 문제 Number Line Jumps 리뷰

https://fromitive.github.io/fromitive-blog/coding-interview/2023-07-08-hackerrank-1/

**Comment by DDONONG** [comment]
캥거루가 귀여워요
